samples/native-image-sample/README.md

```diff
@@ -0,0 +1,42 @@
+# Cloud Trace Client Libraries with Native Image
+
+This application uses the Google Cloud [Trace Client Libraries](https://github.com/googleapis/java-trace) and can be compiled with Native Image.
+
+**Note:** In practice, the Trace Client Libraries are not used directly but through another tool, such as through the [OpenCensus Cloud Trace integration](https://cloud.google.com/trace/docs/setup/java) or through a framework like Spring via [Spring Cloud GCP Trace](https://github.com/spring-cloud/spring-cloud-gcp/blob/master/docs/src/main/asciidoc/trace.adoc).
+
+## Setup Instructions
+
+1. Follow the [GCP Project and Native Image Setup Instructions](../../README.md).
```

Comment:
This link will disappear once we archive the repo. How about we copy over the contents? https://github.com/GoogleCloudPlatform/native-image-support-java/blob/master/native-image-samples/README.md

Reply:
Good catch. Fixed.